src/synom.rs

```diff
@@ -213,6 +213,16 @@ impl Synom for proc_macro2::TokenStream {
     }
 }
 
+impl Synom for proc_macro2::Ident {
+    fn parse(input: Cursor) -> PResult<Self> {
+        input.term().ok_or_else(|| ParseError::new("not an ident"))
```

Comment:
I would like this to preserve the behavior of 0.13's impl Synom for Ident. Separately we will need some way to do https://docs.rs/syn/0.13/syn/struct.Ident.html#method.parse_any.